doc/index.rst

```diff
@@ -48,17 +48,49 @@ For example, the ``'qiskit.aer'`` device with two wires is called like this:
 Backends
 ~~~~~~~~
 
-Qiskit devices have different **backends**, which define which actual simulator or hardware is used by the
-device. Different simulator backends are optimized for different types of circuits. A backend can be defined as
-follows:
+Qiskit devices have different **backends**, which define which actual simulator or hardware is 
+used by the device. We recommend passing in a backend instance rather than a string with the 
+backend name to the devices. 
```

Comment:
```suggestion
used by the device. A backend instance should be and passed to the device when initializing.
```

Since the string method is only really relevant for the `aer` device now, and will potentially be deprecated soon, I think we should just avoid mentioning it entirely in the general index.